lib/Runtime/Library/JavascriptBuiltInFunctionList.h

```diff
@@ -121,7 +121,7 @@ BUILTIN(JavascriptDate, ToLocaleTimeString, EntryToLocaleTimeString, FunctionInf
 BUILTIN(JavascriptDate, ToString, EntryToString, FunctionInfo::ErrorOnNew | FunctionInfo::HasNoSideEffect)
 BUILTIN(JavascriptDate, ToTimeString, EntryToTimeString, FunctionInfo::ErrorOnNew)
 BUILTIN(JavascriptDate, ToUTCString, EntryToUTCString, FunctionInfo::ErrorOnNew)
-BUILTIN(JavascriptDate, ToGMTString, EntryToGMTString, FunctionInfo::ErrorOnNew)
+BUILTIN(JavascriptDate, ToGMTString, EntryToUTCString, FunctionInfo::ErrorOnNew) // Spec stipulates toGMTString must be the same function object as toUTCString
```

Comment:
Thanks for pointing me at those functions. I took a stab at it to get the test to pass and tried to ensure other tests wouldn't break but it appears I missed some.